.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    paths:
+      - 'charts/**'
```

Comment:
can this chart code be moved to `deploy/charts`, which might require updates to github actions or customization. Pushing to `deploy/charts` will help consolidate all the deployment code under one directory.